src/person/actions/add-person.action.ts

```diff
@@ -0,0 +1,21 @@
+import { Fetcher } from '../../common/api';
+import { endpoints } from '../api/endpoints';
+import { PersonErr, errorFactory } from '../errors';
+import { IAddPerson } from '../types';
+
+export const addPerson = async (
```

Comment:
```suggestion
export const createPerson = async (
```

Reprendre les termes du CRUD quand c'est possible.
Renommer les fichiers et autres en conséquence.